source/ranges.tex

```diff
@@ -6517,28 +6507,105 @@
 Equivalent to: \tcode{return x.\exposid{current_} - y.\exposid{current_};}
 \end{itemdescr}
 
-\indexlibrarymember{operator-}{elements_view::iterator}%
+\rSec3[range.elements.sentinel]{Class template \tcode{elements_view::\exposid{sentinel}}}
+
+\indexlibraryglobal{elements_view::sentinel}%
+\begin{codeblock}
+namespace std::ranges {
+  template<class V, size_t N>
```

Comment:
This should be:
```c++
  template<input_range V, size_t N>
    requires view<V> && @\placeholder{has-tuple-element}@<range_value_t<V>, N> &&
             @\placeholder{has-tuple-element}@<remove_reference_t<range_reference_t<V>>, N>
```
to match the *template-head* on the `elements_view` template. (Yes, we've consistently been getting this wrong for quite some time, #3750 fixes all pre-existing instances I'm aware of.)

Reply:
This is a normative change - please open an issue with LWG (for this and all of #3750).

Reply:
I believe this fix is editorial: We attempt a redeclaration of the nested class `elements_view::sentinel`, and that only works if the template-head is the same. The code as presented won't compile, because we can't add members to a class later on. The correct requires-clause is shown with the declaration of elements_view, so it seems obvious what is meant here.

Reply:
I see, fixed.